build/validated-languages.json

```diff
@@ -0,0 +1,30 @@
+[
+    "bs",
```

Comment:
I'm confused. Is this the list of "validated" or "unvalidated" languages? If unvalidated, I'd name this file something like that.